### PR TITLE
Tweak recipes of glasses

### PR DESCRIPTION
**Pull request by m-dzianishchyts**
## What Does This PR Do
Tweaks recipes of glasses to fix some issues. Uses defines for time values.

## Why It's Good For The Game
1. We better forbid using subtypes of a HUD to disable crafting some HUD from another crafted HUD because it consumes resources irrevocably (ok for cables).
2. We better forbid using subtypes of sunglasses because:
    a. Removal recipes return simple sunglasses.
    b. Different sunglasses have different features that HUD sunglasses do not have.

UPD. Came to the conclusion that these are ok:
```dm
/obj/item/clothing/glasses/sunglasses
/obj/item/clothing/glasses/sunglasses/noir
/obj/item/clothing/glasses/sunglasses/yeah
/obj/item/clothing/glasses/sunglasses/big
```

## Testing
Local server ok.

## Changelog
:cl: Maxiemar
tweak: Different HUD sunglasses and HUD goggles recipes allow simple HUDs and only some sunglasses now.
/:cl:

### CODE REVIEW

**Comment by FunnyMan3595** [comment]
I'd like to see a full list of the objects we're blacklisting. It's not obvious to me that *all* subtypes don't make sense.

**Comment by FunnyMan3595** [comment]
Hmm, the blindfolds, laser, and reagent versions all seem fine to remove. Prescription as well, maybe, since they have another use. But the rest all seem OK to me.

File: code/modules/crafting/tailoring.dm [hunk 1]
```diff
@@ -1,53 +1,60 @@
...
-	time = 40
+	time = 4 SECONDS
 	category = CAT_CLOTHING
 
 /datum/crafting_recipe/durathread_bandana
 	name = "Durathread Bandana"
 	result = list(/obj/item/clothing/mask/bandana/durathread)
 	reqs = list(/obj/item/stack/sheet/durathread = 1)
-	time = 25
+	time = 3 SECONDS
```
Comment (Henri215) [inline code/modules/crafting/tailoring.dm:49]: 2.5 seconds --> 3, 0.5 seconds --> 1, 4.5 seconds --> 5.
You changed the time on some recipes without mentioning on the description, if you wish to standardize/tweak the times to build then it would be better for another PR since this one is focused on glasses.

**Review by DGamerL — APPROVED** [review]
